crates/ef-testing/Cargo.toml

```diff
@@ -8,6 +8,7 @@ repository.workspace = true
 build = "build.rs"
 
 [dependencies]
+build-utils = { path = "../build-utils" }
```

Comment:
are we using this?

Reply:
actually, it can stay as a build util

Reply:
I'm not sure I understand, do you need to import this in ef-testing?

Reply:
yes - but it was already importer as a build-dependency so I will leave it as one